src/components/SecondaryPanes/index.js

```diff
@@ -129,6 +129,7 @@ class SecondaryPanes extends Component<Props> {
 
     return {
       header: L10N.getStr("scopes.header"),
+      cssClass: "scopes-pane",
```

Comment:
can we use `className` or is that reserved?

Reply:
The original version I made had className, but it felt confusing to use className property to pass values to className as we would have the same name in two places doing two different things.